### DISCUSSION TITLE
Create a Pointer from an address.

### DISCUSSION THREAD

**Discussion by BerezkovN** [bytedeco/javacpp#620] [Q&A] [answered]
Hello! I have a memory block allocated with MemoryUtil.nmemAlloc for an RGBA image. I need to convert it to YUV420p with sws_scale, but some parameters require IntPointer. Is there a correct way to convert address(long) to pointer(IntPointer)?

**Comment by saudet** [comment, marked as answer]
Create your own subclass of IntPointer, possibly like this: `new IntPointer() {{ address = myAddress; }}`.
